soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_progress.c

```diff
@@ -0,0 +1,456 @@
+    gGanonsCastleTitleCardFRATex,    gBottomOfTheWellTitleCardFRATex,  gIceCavernTitleCardFRATex,
+    gGERudoTrainingGroundTitleCardFRATex,
+};
+//Dungeon tokens
+Kaleido_sprites ForestT_Kspr = { gForestMedallionIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites FireT_Kspr = { gFireMedallionIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites WaterT_Kspr = { gWaterMedallionIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites SpritT_Kspr = { gSpiritMedallionIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites ShadowT_Kspr = { gShadowMedallionIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites LightT_Kspr = { gLightMedallionIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites KokiriT_Kspr = { gKokiriEmeraldIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites GoronT_Kspr = { gGoronRubyIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites ZoraT_Kspr = { gZoraSapphireIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+Kaleido_sprites GymT_Kspr = { gGerudosCardIconTex, 24, 24, 0.5f, 0.5f, 0, 0, G_IM_FMT_RGBA, { 255,255,255,255 }, false };
+
+/**
+ * Sound effects
+ */
+s32 Close_SFX = NA_SE_SY_DECIDE;
+
+/**
+ * Settings
+ */
+s16 FrameSinceOpen = 0; //Count the frame since this menu is open. (used to make a little latence before being able to close it.)
+u8 MapIndex_GSFlags[] = { //Array used to detect when ever a player has all gs token from a dungeon.
+    0x0F, 0x1F, 0x0F, 0x1F, 0x1F, 0x1F, 0x1F, 0x1F, 0x07, 0x07, 0x03,
+    0x0F, 0x07, 0x0F, 0x0F, 0xFF, 0xFF, 0xFF, 0x1F, 0x0F, 0x03, 0x0F,
+};
+s16 X_Offset_GS[3][13] = {//same order than TitleCard[], this i sthe X position of GS icon next to dungeon label. 
+    {-6,-6,-6,
+     -4, 2,-2,
+     -1,-5,-3,
+     -2,-6, 4,
+     -8},//ENG.
+
+    {-3,-5,-5,
+      1, 2,-2,
+     -2,-4,-5,
+     -3,-6, 8,
+     -2},//GER.
+
+    {1,-6,-6,
+    -5,-2,-4,
+    -5,-4,-5,
+    -5,13,-6,
+    -6},//FRA.
+};
+
+void MenuBackground_Draw(GlobalContext* globalCtx, Kaleido_sprites* sprite, s16 line, s16 col) {
+    InterfaceContext* interfaceCtx = &globalCtx->interfaceCtx;
+    s16 alpha = sprite->color.a;
+
+    s16 Size_W = BG_KAL_PROG_W; //Used to ajust script to the last image.
+    s16 Size_H = BG_KAL_PROG_H;
+    if(col == 3) {
+        Size_W = BG_KAL_PROG_W_L;
+    }
+
+    int height = Size_H * sprite->scale_h;
+    int width = Size_W * sprite->scale_w;
+
+    int width_factor = (1 << 10) * Size_W / width;
+    int height_factor = (1 << 10) * Size_H / height;
+
+    OPEN_DISPS(globalCtx->state.gfxCtx, "z_kaleido_progress.c", 51);
```

Comment:
```suggestion
    OPEN_DISPS(globalCtx->state.gfxCtx, __FILE__, __LINE__);
```
Same for close disps and anywhere else this comes up

Reply:
WOW Thanks for that that much better !